app/views/validation.scala.html

```diff
@@ -67,9 +67,9 @@
             <div id="svv-panorama-holder">
                 <div id="svv-panorama-outline">
                     <div id="svv-panorama">
-                        <div id="view-control-layer" style="cursor: url(/assets/javascripts/SVLabel/img/cursors/openhand.cur) 4 4, move; z-index: 2;">
+                        <div id="view-control-layer" style="cursor: url(/assets/javascripts/SVLabel/img/cursors/openhand.cur) 4 4, move; z-index: auto;">
                             <button id="label-visibility-button-on-pano" class="button label-visibility-button-on-pano" title="@Messages("press.key", "H")" data-toggle="tooltip" data-placement="top">@Html(Messages("validate.top.ui.hide.label"))</button>
-                            <div id="label-description-box"></div>
+                            <div id="label-description-box" style="z-index: 2;"></div>
```

Comment:
Why is this update of the z-index necessary? Were either of these updates to the z-index necessary? We don't need these to make the arrows clickable, so not sure why we would need them now.

Reply:
Hey, most of the other code was necessary but removing these z-index changes makes the links unclickable again on the validation page. I'd prefer to keep these here.

Reply:
Okay I just tested and it looks like messing with the z-index is also messing with the cursor. It should be an open-hand cursor on the validation page, but with these changes it goes back to the default cursor. The z-index change is also allowing us to move in GSV using the arrow keys, which we specifically tried to disallow (probably by adding this layer with a higher z-index to cover GSV).

 This is all making me think that we used that method of making things clickable on the audit page for a reason. Maybe it's best to implement something similar to that on the validation page?